src/pytest_cpp/helpers.py

```diff
@@ -0,0 +1,8 @@
+
+from typing import Sequence
+
+
+def make_cmdline(harness: Sequence[str], executable: str, arg: Sequence[str] = ()) ->Sequence[str]:
+    if not executable:
```

Comment:
Hmm why is this needed?

Reply:
ohh, I'm C++ developer much more than Python :)

Reply:
Hehehe OK we can drop the if then, I think.